app/models/passenger.rb

```diff
@@ -0,0 +1,19 @@
+class Passenger < ApplicationRecord
+  has_many :trips, dependent: :nullify
+
+  def total_charged
+    trips_with_passenger_id = self.trips.where(passenger_id: id)
```

Comment:
The `where` here is redundant since the `trips` method created by the `has_many` relationship will only return trips for this passenger:

```suggestion
    trips_with_passenger_id = self.trips
```